backend/webhooks/events/events.go

```diff
@@ -0,0 +1,44 @@
+package events
+
+import "github.com/teamhanko/hanko/backend/persistence/models"
+
+type Event string
+
+const (
+	User         Event = "user"
+	UserCreate   Event = "user.create"
+	UserUpdate   Event = "user.update"
+	UserDelete   Event = "user.delete"
+	Email        Event = "user.update.email"
```

Comment:
This event is only used once (while checking if an event is valid), so it should be removed.